packages/cli/src/services/pruning/__tests__/pruning.service.test.ts

```diff
@@ -8,9 +8,13 @@ import { mockLogger } from '@test/mocking';
 
 import { PruningService } from '../pruning.service';
 
+jest.mock('@/db', () => ({
+	connectionState: { migrated: true },
+}));
+
 describe('PruningService', () => {
 	describe('init', () => {
-		it('should start pruning if leader', () => {
+		it('should start pruning if leader main', () => {
```

Comment:
This sounds a bit weird. Maybe it could be

```suggestion
		it('should start pruning on main instance that is the leader', () => {
```